sdk/azcore/CHANGELOG.md

```diff
@@ -1,13 +1,36 @@
 # Release History
 
+## v0.19.0
+
+### Breaking Changes
+* Split content out of `azcore` into various packages.  The intent is to separate content based on its usage (common, uncommon, SDK authors).
+  * `azcore` has all core functionality.
+  * `log` contains facilities for configuring in-box logging.
+  * `policy` is used for configuring pipeline options and creating custom pipeline policies.
+  * `runtime` contains various helpers used by SDK authors and generated content.
+  * `streaming` has helpers for streaming IO operations.
```

Comment:
I'm not totally in love with the name `streaming`, so if anybody has a better idea please post them here.  `io` would really be ideal by that collides with the standard library.